docs/tutorials/sending-and-receiving-messages-with-mqtt.md

```diff
@@ -94,7 +95,8 @@ test it on our PC or on another EV3:
       client.subscribe("topic/test")
 
     def on_message(client, userdata, msg):
-      if (msg.payload == "Hello world!"):
+      # Convert payload from binary to ascii-format
```

Comment:
Don't you mean "from ascii"? Strings in python3 are always unicode (by default UTF-8 encoding). So, `b'string'.decode('ascii')` would decode the binary data `b'string'`, interpreting it as ascii data and return a python string (UTF-8 encoded).